clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -107,6 +108,7 @@ class IoUtil(storage: Storage) {
   }
 
   def writeGoogleObjectData(data: String, location: URI): Unit = {
+    logger.info(s"Writing to '$location'...")
```

Comment:
To me this seems more like DEBUG than INFO. Do we have a policy on what is INFO vs DEBUG?

Reply:
We don't have a good policy on that. I was trying to mimic the output that `gsutil` used to spit out.